### DISCUSSION TITLE
Timezone for timestamps

### DISCUSSION THREAD

**Discussion by sdauth** [grafana/grafana-csv-datasource#78] [Q&A] [answered]
Hello,

i am trying to visualize a time-series from a csv-file
the timestamps in the file are in the format YYYY-MM-DD HH:mm:ss, given in my local time (UTC+2)

if i set the data-type of my 'time' field in the query-editor to "time", the timestamp is converted to +2 hours
for example: 2021-05-29 22:02:35 => 2021-05-30 00:02:35

if i change the timezone in grafanas timerange-selection to utc, only data older than 2 hours is displayed.

it seems changing the datetime-format to YYYY-MM-DD HH:mm:ss+hh:ss doesn't change the behavior.

is there any possibility to specify the timezone for the data to display the correct time?

kind regards,
Simon

**Comment by marcusolsson** [comment, marked as answer]
Hi,

Since the timestamps are parsed on the server, if they have no timestamp information, then UTC is assumed.

Explicitly setting the default time zone is not supported right now. If this is important to you, please consider creating a feature request.

**Reply by sdauth** [reply]
Hi,

thanks for your reply.
in the meantime, i found out, that i was missing a space in my Datetime-Format with timestamp

that way, it works correctly
YYYY-MM-DD HH:mm:ss +hh:ss

kind regards,
Simon
